### PR TITLE
Team 2016

### PR DESCRIPTION
**Pull request by afreemantle**
2015-2016 team page updated

### CODE REVIEW

**Comment by kevin1** [comment]
One thing to be careful of -- please don't replace the existing photos. This will also cause their photo to change on previous years' team pages. There are some examples in existing photos, e.g. frank-fang.jpg and frank-fang-2.jpg

**Comment by kevin1** [comment]
Also, it looks like the photos have the execute bit set (permissions are 755 rather than 644). Could you do `chmod -x *.jpg` in each directory?

File: team/team.2016.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+{% include team-member.html name="Bruce Lee" caption="Intake" img="2015/bruce-lee.jpg" %}
+{% include team-member.html name="Alex Malouf" caption="Marketing & Brand" img="2018/alex-malouf.jpg" %}
+{% include team-member.html name="Manjul Pahwa" caption="Brakes" img="2018/manjul-pahwa.jpg" %}
+{% include team-member.html name="Tom Rasmussen" caption="Hubs & Uprights" img="2017/tom-rasmussen.jpg" %}
+{% include team-member.html name="Matt Sheridan" caption="Suspension" img="2016/matt-sheridan.jpg" %}
+{% include team-member.html name="Christine Sison" caption="Brakes" img="2018/christine-sison.jpg" %}
+{% include team-member.html name="Diego Song" caption="Electric Racecar" img="2016/diego-song.jpg" %}
+{% include team-member.html name="Pierre Suignard" caption="Sponsorship" img="2018/pierre-suignard.jpg" %}
+{% include team-member.html name="Nicho Villalobos" caption="Aerodynamics" img="2018/nicho-villalobos.jpg" %}
+{% include team-member.html name="Yumou “William” Wei" caption="Body & Steering" img="2017/yumou-wei.jpg" %}
```
Comment (kevin1) [inline team/team.2016.md:61]: How are these names ordered? (In the past we've done alphabetical by last name)

Comment (afreemantle) [inline team/team.2016.md:61]: Ahhh, I didn't notice the ordering at first. I will order by last name. Should I reset head back to before I replaced existing photos and then make the changes you have requested above?

Comment (kevin1) [inline team/team.2016.md:61]: yeah, that'd probably be the easiest way

**Comment by afreemantle** [comment]
Hey Kevin, how does this look now?

**Comment by kevin1** [comment]
good work, merged

**Pull request merged**